dimod/serialization/bson.py

```diff
@@ -13,31 +13,43 @@
 #    limitations under the License.
 #
 # ================================================================================================
+from __future__ import absolute_import
 import itertools
 
 import numpy as np
 
 from dimod.binary_quadratic_model import BinaryQuadraticModel
 
+from six import PY2
+
 
 def bqm_bson_encoder(bqm):
     """todo"""
+
+    if PY2:
```

Comment:
Ok, I see what this is trying to do, but it definitely needs at least a comment explaining